client-report/package.json

```diff
@@ -4,83 +4,61 @@
   "description": "pol.is report",
   "scripts": {
     "clean": "rimraf dist",
-    "build:webpack": "NODE_ENV=production webpack --config webpack.config.js",
-    "build": "npm run clean && npm run build:webpack",
-    "deploy:preprod": "gulp deployPreprod",
-    "deploy:prod": "gulp deploy_TO_PRODUCTION",
+    "bundle:prod": "webpack --mode=production",
+    "bundle": "npm run clean && npm run build:prod",
+    "build:prod": "gulp build",
```

Comment:
Let's completely nuke gulp!

Reply:
This will be covered when https://github.com/compdemocracy/polis/issues/90 is done.

Also worth looking at `polis-whitelabel` solution
https://mathdem.slack.com/archives/C0374J43QMT/p1679306674336269?thread_ts=1678786732.540639&cid=C0374J43QMT